test/common/router/rds_impl_test.cc

```diff
@@ -290,6 +290,10 @@ TEST_F(RdsImplTest, Basic) {
   // Old config use count should be 1 now.
   EXPECT_EQ(1, config.use_count());
   EXPECT_EQ(2UL, scope_.counter("foo.rds.foo_route_config.config_reload").value());
+  EXPECT_NE(0, scope_
```

Comment:
could this flake if the test runs in <1ms?

If you are just trying to ensure that the gauge exists, you can use findGaugeByString, which is on class TestStore, which is the base class of MockIsolatedStatsStore.

Reply:
Sure.. Will change to ensure gauge exists.